autoload/rainbow_csv.vim

```diff
@@ -883,12 +892,13 @@ func! rainbow_csv#csv_align()
         let is_first_line = 0
     endfor
     if !has_edit
-        echoerr "File is already aligned"
+        echoerr "Range is already aligned"
     endif
 endfunc
 
-
-func! rainbow_csv#csv_shrink()
+func! rainbow_csv#csv_shrink() range
```

Comment:
All my comments regarding the align function also apply to Shrink function.

Reply:
done.
moved the show_progress_bar setting to location similar to that found in _align, so that the exception processing block for md/rmd files could remain in a single location, as in _align.